youtube_dl/extractor/rtp.py

```diff
@@ -3,56 +3,100 @@
 
 from .common import InfoExtractor
 from ..utils import (
+    ExtractorError,
     determine_ext,
     js_to_json,
 )
+from ..compat import (
+    compat_b64decode,
+    compat_urllib_parse_unquote,
+)
+
+import re
+
+
+def decode_b64_url(code):
+    decoded_url = re.match(r"[^[]*\[([^]]*)\]", code).groups()[0]
+    return compat_b64decode(
+        compat_urllib_parse_unquote(
+            decoded_url.replace('"', '').replace('\'', '').replace(',', ''))).decode('utf-8')
```

Comment:
Maybe simplify with:
```
            decoded_url.translate(None, '"\',').decode('utf-8')
```
Or in case there could be spaces in `['"cod", "ed ", "URL"]`:
```
            re.sub(r'[\s"\',]', '', decoded_url).decode('utf-8')
```